commit 3cfa937ecdf642e933e38ce910c846ef14e2773c (refs/changes/96/396/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-09-01T15:14:02-07:00 (3 years, 1 month ago)
    
    RELENG-715 ironically the "ENG" jira project is not an engineering project

diff --git a/tools/jira/listengprojects.sh b/tools/jira/listengprojects.sh
index b54aa8a..2ab1e3d 100755
--- a/tools/jira/listengprojects.sh
+++ b/tools/jira/listengprojects.sh
@@ -46,7 +46,6 @@ DATASET
 DCAPI
 DOC
 DOCKER
-ENG
 FWAPI
 HEAD
 HVM
